lib/tasks/data.rake

```diff
@@ -42,10 +39,32 @@ namespace :data do
     ) do |row|
       data << row.to_h
     end
+    data
+  end
+
+  def import_csv(file_name, klass)
+    puts "Import started for #{klass}"
+    current_time = Time.now
+    data = csv_data(file_name)
     result = klass.import(data, validate: false)
     puts "Imported #{result.num_inserts} records for #{klass} in #{Time.now - current_time} seconds"
   end
 
+  def import_conf_and_speaker
+    conferences = csv_data('conferences')
+    speakers = csv_data('speakers')
+    ticket_prices = csv_data('ticket_prices')
+    current_time = Time.now
+
+    conferences.each_with_index do |c, i|
+      conf = Conference.create(c)
+      conf.speakers << Speaker.create(speakers[i])
+      conf.ticket_prices.create(ticket_prices)
```

Comment:
This is like same ticket price for all the conferences right? but each conference can have different ticket prices